tasks/arena-deploy.ts

```diff
@@ -0,0 +1,229 @@
+import { task, types } from 'hardhat/config';
```

Comment:
@henryacaron Is this file a straight copy of `deploy.ts`?

Reply:
yes except the deployFacet lines are all different and we agreed not to overwrite any code